setup.sh

```diff
@@ -8,19 +8,15 @@ echo 'Installing macOS packages...'
 brew update
 brew install node
 brew install mongodb
-brew install grunt
```

Comment:
Even though [Grunt is technically in Homebrew's repositories](https://github.com/Homebrew/homebrew-core/blob/master/Formula/grunt-cli.rb), I can't find anywhere in the [Grunt documentation](https://gruntjs.com/installing-grunt) about installing it via Homebrew. It seems the reccomended way is through npm, so I think we might want to add `yarn global add grunt-cli` instead.